### PR TITLE
Sleepy Cow Sorting Java Solution

### PR DESCRIPTION
**Pull request by brad182**

### CODE REVIEW

File: solutions/usaco-892.mdx [hunk 1]
```diff
@@ -38,5 +38,41 @@ file_out.close()
...
+			cows[i] = io.nextInt();
+		}
+
+		// Find the number of strictly increasing values at the end of the list
+		int answer = n - 1;
+		for (int i = n - 2; i >= 0; i--) {
+			if (cows[i] < cows[i + 1]) {
+				answer = i;
+			}
+			else {
```
Comment (SansPapyrus683) [inline solutions/usaco-892.mdx:64]:
```suggestion
			} else {
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
lmao you're probably going to reject any other change i make so i'm not going to bother

File: solutions/usaco-892.mdx (hunk 1 above)
Comment (brad182) [reply solutions/usaco-892.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]
yeah alr i guess it's good enough to read

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**